lib/api/multiObjectDelete.js

```diff
@@ -271,9 +272,27 @@ export function getObjMetadataAndDelete(authInfo, canonicalID, request,
             // TODO: update number of objects to reflect creation of
             // delete markers; Utapi must be updated to allow this for
             // 'deleteObject' metrics
+            let isDeleteMarker;
+            let deleteMarker;
+            // If trying to delete an object that does not exist (empty objMD)
+            // return DeleteMarkerVersionId equals the versionID generated and
+            // DeleteMarker equals true
+            if (!objMD) {
```

Comment:
isn't this the case not just if we create a delete marker for an object that doesn't exist, but in general if we create a delete marker? I think this is why I created the variable `deleteInfo.newDeleteMarker` and set it equal to true if we create a delete marker. In that case you can replace `!objMD` here with `deleteInfo.newDeleteMarker`.